api/HTMLAudioElement.json

```diff
@@ -112,11 +112,11 @@
             },
             "firefox": {
               "version_added": "4",
-              "version_removed": "31"
+              "version_removed": "28"
```

Comment:
This has been gone for long enough now that it can be removed from BCD. https://developer.mozilla.org/docs/Web/API/HTMLAudioElement/mozCurrentSampleOffset is 404 too. But fixing the data what we're up to here, and that's OK.